signatures/rainbow/rainbow/bench/opt_IIIc_skylake.yaml

```diff
@@ -0,0 +1,10 @@
+platform: Intel iXEON E3-1275 (Skylake) 3.6GHZ
+timings:
+  unit: cycles
+  gen: 137000000
+  enc|sign: 87200000
+  dec|vrfy: 43000000
```

Comment:
One of us must have mixed sth up here. I find 94.8M / 588K / 114K

Reply:
Apparently I messed up; totally agree with your numbers.